packages/fuels-types/src/param_types.rs

```diff
@@ -292,8 +290,19 @@ fn try_struct(the_type: &Type) -> Result<Option<ParamType>, Error> {
     Ok(result)
 }
 
+fn has_struct_format(field: &str) -> bool {
+    field.starts_with("struct ")
+}
```

Comment:
Why isn't it in the `fuels-abi-types` utils?

Reply:
`fuel-abi-types` *

We only put what we absolutely had to in the `fuel-abi-types` . Technically we no longer have sole ownership over that project so changes we might want to do will have to be checked against all the users of the library.

Also, changes to `fuel-abi-types`, breaking or not, have to go through a release to be made available to `fuels-rs`, which causes delays.

So:
1. If we put it there, we have to be careful while changing it.
2. Nobody is using it except us, no need to share.
3. Anything put there takes time to modify.

Also, we'll probably talk to the compiler team about saving the fn selectors in the json abi. If we manage to get that through, then this will be deleted.

Reply:
OOOH saving the fn selectors in the JSON ABI sounds super smart lol. 
I get it, but I feel like any library that would like to work over the `fuel-abi-types` would need the helpers function we use? For example someone writing another SDK? Don't we want to make this available? 
But I agree this would be a larger discussion.